ERCS/erc-7630.md

```diff
@@ -0,0 +1,123 @@
+---
+eip: 7630
```

Comment:
```suggestion
eip: 7631
```
Numbers are assigned by ERC editors/associates. Please don't self assign. 
To avoid any perception of number sniping, I am assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
Sorry. Was simply trying to find the next number and was off-by-one. Thanks!

Reply:
When authors self assign I manually renumber to avoid any perception of number sniping.